phpstan-baseline.neon

```diff
@@ -4,4 +4,6 @@ parameters:
         - src
     tmpDir: build/phpstan
     checkOctaneCompatibility: true
-    checkModelProperties: true
\ No newline at end of file
+    checkModelProperties: true
+    ignoreErrors:
+        - '#Access to an undefined property .*::\$parser#'
```

Comment:
Stan complains because parser of course doesn't exist on the TestComponent/View/Response classes, but we're adding it dynamically.